### ISSUE TITLE
flow: allow to include parameters in redirects

### ISSUE THREAD

**Issue by x1B** [LaxarJS/laxar#417]
When a place redirects to another place, there is currently no way to set parameter values.
This is a must-have for LaxarJS v1 feature parity.

Redirects should either
 - be expressed as URLs or,
 - allow to specify a placeId together with set of parameter (default) values.

Currently, parameters from the source place are forwarded to the destination place. However, such parameters will not always be available (e.g. an "index" place with pattern "/" might want to redirect a user to "/pages/:pageId" with "pageId" set to "landing").

In the future (issue TBD) we *might* want to support some way of programmatic redirects. However, this can already be achieved
 - through server configuration,
 - through programmatic navigation (e.g. generating `navigateRequest` from a dispatcher page).

**Comment by alex3683** [comment]
The solution we agreed upon: We'll currently leave the `redirectTo` property untouched (i.e. as described above) and instead add another place property called `redirectToPath`. This option, as the name already states, expects a complete path to navigate to, starting with `/` and including the values for all relevant place parameters. Values for parameters of the place redirecting from will not be mixed in.
By taking this road we offer an explicit way to define static redirects without changing the way the "normal" `redirectTo` currently works.

**Comment by alex3683** [comment]
Implemented on `master` (`v2.0.0`)

### Upgrade Information

As alternative to `redirectTo`, a place may now specify a complete path including all parameter values to redirect to by using the `redirectToPath` property of a place.

Example:
```JSON
{
   "places": {
      "details": {
         "patterns": [ "/details/:item", "/details" ],
         "page": "first_page"
      },

      "team": {
         "patterns": [ "/team" ],
         "redirectToPath": "/details/team"
      }
   }
}
```
